src/test/scala/sparkz/core/api/http/PeersApiRouteSpec.scala

```diff
@@ -38,7 +38,7 @@ class PeersApiRouteSpec extends AnyFlatSpec
   private val body = HttpEntity("localhost:8080".asJson.toString).withContentType(ContentTypes.`application/json`)
   private val badBody = HttpEntity("badBodyContent".asJson.toString).withContentType(ContentTypes.`application/json`)
 
-  private val restApiSettingsWithApiKey = RESTApiSettings(addr, Some(BCrypt.hashpw(credentials.password(), BCrypt.gensalt())), None, 10 seconds)
+  private val restApiSettingsWithApiKey = RESTApiSettings(addr, Some(BCrypt.`with`(BCrypt.Version.VERSION_2Y).hashToString(12, credentials.password().toCharArray)), None, 10 seconds)
```

Comment:
what is the meaning of `12` ?

Reply:
It is the cost factor of the algorithm which is suggested to be >= 10

Reply:
let's add a `val factor = 12` with some comments what is it.
then use it in this test